pkg/dockerutil/dockerutils.go

```diff
@@ -1904,3 +1904,19 @@ func GetContainerNames(containers []dockerTypes.Container) []string {
 func IsErrNotFound(err error) bool {
 	return dockerClient.IsErrNotFound(err)
 }
+
+// CanRunWithoutDocker returns true if the command or flag can run without Docker.
+func CanRunWithoutDocker() bool {
+	if len(os.Args) < 2 {
+		return true
+	}
+	commands := map[string]bool{
```

Comment:
There are some interesting variants that I think we can ignore, like `ddev help -h` and `ddev -h help`

And what about all the things like `ddev config -h` and `ddev config --help` ?

Reply:
> what about all the things like `ddev config -h` and `ddev config --help` ?

I added support for Cobra help flag.